usr/local/share/bastille/common.sh

```diff
@@ -113,7 +113,7 @@ generate_static_mac() {
     local jail_name="${1}"
     local external_interface="${2}"
     local external_interface_mac="$(ifconfig ${external_interface} | grep ether | awk '{print $2}')"
-    local macaddr_prefix="$(echo ${external_interface_mac} | cut -d':' -f1-3)"
+    local macaddr_prefix="58:9c:fc"
     local macaddr_suffix="$(echo -n "${external_interface_mac}${jail_name}" | sed 's#:##g' | sha256 | cut -b -5 | sed 's/\([0-9a-fA-F][0-9a-fA-F]\)\([0-9a-fA-F][0-9a-fA-F]\)\([0-9a-fA-F]\)/\1:\2:\3/')"
     if [ -z "${macaddr_prefix}" ] || [ -z "${macaddr_suffix}" ]; then
```

Comment:
It is worth looking, synchronizing with vm-bhyve here: https://github.com/churchers/vm-bhyve/blob/dc92eeaae983bef284eaa920f1a1b2aed33b3767/lib/vm-run#L905-L934

Reply:
I will have a look again tomorrow, As fas as I understand you want it to be reproducible, then you cannot you the time for that, of course.

Reply:
Worth looking at `/usr/src/sys/net/ieee_oui.h`

Reply:
I would look at `/usr/src/sys/net/if_ethersubr.c`: `ether_gen_addr_byname(const char *nameunit, struct ether_addr *hwaddr)`. This is exactly what you are looking for.